### ISSUE TITLE
403 error when trying to get token

### ISSUE THREAD

**Issue by aronkvh** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#78]
I get this error when trying to login to access the token.
what am I doing wrong?

```
Hitelesítési hiba
Error (403): restricted_client
This app is not yet configured to make OAuth requests. To do that, set up the app’s OAuth consent screen in the Google Cloud Console https://console.developers.google.com/apis/credentials/consent?project=${your_project_number}
A kérelem adatai

    access_type=offline
    response_type=code
    redirect_uri=urn:ietf:wg:oauth:2.0:oob
    client_id=673272552947-rsjbe096k1uu1lgurqslshal6804j800.apps.googleusercontent.com
    scope=https://www.googleapis.com/auth/assistant-sdk-prototype

```
 - OS: [KDE Neon focal]

**Comment by Melvin-Abraham** [comment]
@aronkvh You seem to have skipped configuring the consent screen. You can get help with setting up consent screen using [this link](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client#configure-consent-screen)

**Comment by aronkvh** [comment]
Thanks, I just configured it with the wrong account

**Comment by TheCrazyMax** [comment]
Having followed [https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client#configure-consent-screen](this link) to set up the OAuth Consent Screen, I still a 403 error and I'm starting to go crazy on this.
I saw people online saying they had to set an app logo for it to work. It doesn't work either on my case. It seems there is no way I can get that to work, I tried solutions found online for several hours, but nothing is working.
